PROTO_tests/tests/dtypes_test.py

```diff
@@ -27,8 +27,9 @@ class TestDTypes:
     def test_check_np_dtype(self, dtype):
         dtypes.check_np_dtype(np.dtype(dtype))
 
-    @pytest.mark.parametrize("dtype", [np.dtype(np.int16), "np.str", ak.bigint])
+    @pytest.mark.parametrize("dtype", ["np.str", ak.bigint])
     def test_check_np_dtype_errors(self, dtype):
+        print(dtype)
```

Comment:
Leftover print statement

Reply:
Thanks for catching this.  I removed it.